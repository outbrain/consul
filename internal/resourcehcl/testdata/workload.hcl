ID {
  Type = gvk("catalog.v2beta1.Workload")
  Name = "work1"
}

Data {
  Addresses = [
    {
      Host = "127.0.0.1"
    },
  ]
  Ports = {
    "http" = {
      port = 8080
    }
  }
}

Metadata = {
  "foo" = "bar"
}