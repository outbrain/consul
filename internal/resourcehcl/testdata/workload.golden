{"id":{"name":"work1", "type":{"group":"catalog", "groupVersion":"v2beta1", "kind":"Workload"}}, "metadata":{"foo":"bar"}, "data":{"@type":"hashicorp.consul.catalog.v2beta1.Workload", "addresses":[{"host":"127.0.0.1"}], "ports":{"http":{"port":8080}}}}